polytracker/taint_dag.py

```diff
@@ -195,7 +196,7 @@ def read_sink(self, offset: int) -> TDSink:
         return result
 
     @property
-    def sinks(self) -> Generator[TDSink, None, None]:
+    def sinks(self) -> Iterator[TDSink]:
         assert self.header.sink_mapping_offset > 0
```

Comment:
Suggest that we move the assertions to the __init__-function to get a early error if there is something wrong with the tdag-file.